fields/types/list/ListType.js

```diff
@@ -101,7 +103,7 @@ list.prototype.addToSchema = function () {
 			);
 		}
 		fields[path] = createField(path, fieldsSpec[path]);
-		fieldsArray.push(path);
+		paths.push(path);
```

Comment:
paths means something specific on Field Types - it's a nested set of strings you can pull values from when processing incoming data (or use to generate the names of form fields)

neither of those will work in this case, which is why I mirrored the `fieldsArray` concept from the main `List` class